tests/test_other.py

```diff
@@ -2151,7 +2151,7 @@ def test_llvm_nativizer(self):
     open(os.path.join(self.get_dir(), 'test.file'), 'w').write('''ay file..............,,,,,,,,,,,,,,''')
     open(os.path.join(self.get_dir(), 'stdin'), 'w').write('''inter-active''')
     subprocess.check_call([PYTHON, EMCC, os.path.join(self.get_dir(), 'files.cpp'), '-c'])
-    run_process([PYTHON, path_from_root('tools', 'nativize_llvm.py'), os.path.join(self.get_dir(), 'files.o')], stdout=PIPE)
```

Comment:
getting rid of the stdout redirection makes sense, but doesn't `run_process` also check the return code? we usually prefer that one over the subprocess methods. no big deal either way though.